### ISSUE TITLE
cannot include the header file

### ISSUE THREAD

**Issue by DjMixMasterDragon** [microsoft/vscode-cpptools#1278]
Linux Fedora 25
Vscode Version 1.18.1
C/C++ Ext 0.14.2

All my header include files are squiggled with the error: cannot open source file "... .h"

I have ran the command: gcc -v -E -x c++ - < /dev/null  to confirm those are all in my c_cpp_properties.json and even added a path to stddef.h

Anymore troubleshooting on this case?

**Comment by sean-mcmanus** [comment]
When you click the lightbulb code action do you get any suggested paths (if so, they're obtained via searching the browse.path recursively). What header can't be found? Is it a library header? Also, the order of paths in includePath is important.

**Comment by DjMixMasterDragon** [comment]
When clicking lightbulb, i get a bunch of Add to "include Path"  and Edit "Include Path Settings". 
Yes correct, it is a self defined header file.

**Comment by sean-mcmanus** [comment]
Can you try using the `Add to "includePath"` options, assuming the paths they add are correct and not redundant. By self-defined header file, you mean you created it yourself? Paths in your workspace might not be shown with the gcc command (I think that only shows the system includes).

**Comment by bslote** [comment]
Same problem (I think?) here.

Windows 10
vscode 1.19.0-insider
C/C++ 0.14.3

Header that can't be found is a user-defined header that lives right next to the .cpp file I'm trying to edit. I've tried relative paths and absolute paths to both the directory containing the .h, and to the .h file explicitly. I've tried forward slashes and double backslashes, ${workspaceFolder}/${workspaceRoot} variable substitution and explicit paths, and nothing works.

Additionally I'm not getting an `Add to "includePath"` lightbulb option, there's only an option to `Edit "includePath" setting`.
